modules/displayBook.js

```diff
@@ -0,0 +1,13 @@
+export default function displayBook(book) {
+  const newAddDiv = document.createElement('div');
+  const booksContainer = document.querySelector('.displayBook');
+  newAddDiv.classList.add('bookItem');
+  newAddDiv.id = book.id;
+  newAddDiv.innerHTML = `<ul class="book-content">
+                                <li class = "title">${book.title}</li>
+                                <li>${book.author}</li>
+                            </ul>
+                            <button id="remove">Remove</button>
+    `;
+  booksContainer.appendChild(newAddDiv);
+}
```

Comment:
- [ ] according to the requirements you should use ES6 syntax for this milestone, which means that all functions you created should be arrow functions, which in this case might look something like below:
```
const displayBook = (book) => {
  const newAddDiv = document.createElement('div');
  const booksContainer = document.querySelector('.displayBook');
  newAddDiv.classList.add('bookItem');
  newAddDiv.id = book.id;
  newAddDiv.innerHTML = `<ul class="book-content">
                                <li class = "title">${book.title}</li>
                                <li>${book.author}</li>
                            </ul>
                            <button id="remove">Remove</button>
    `;
  booksContainer.appendChild(newAddDiv);
};

export default displayBook;
```